site/templates/collection/carousel.jet

```diff
@@ -23,5 +23,8 @@
 		<a class="right s72-carousel-control"><span class="fa fa-angle-right"></span></a>
 	</section>
 </s72-carousel>
+{{else}}
+<div class="no-carousel-padding">
```

Comment:
Smit suggested we should indent this. The carousel component isnt indented in the upper section so i left as is for consistency. should they both be indented ideally inside the if / else statements?

Reply:
Yeah they should both be indented!